README.rst

```diff
@@ -1,102 +1,208 @@
 Klister
 ------------
 
-This repository contains a partial implementation of a macro expander
-in which macros can get *stuck*. The idea is that if a macro depends
-on information that will be the result of later macro expansion or
-type checking, it can pause its execution until that information has
-become available.
-
-For a description of one way in which stuck macros might be useful,
-please see this `talk`_.
-
-.. _talk: https://www.youtube.com/watch?v=nUvKoG_V_U0
-
-
-Installation
-============
-
-Klister is written in Haskell, and is built with HPack and Cabal. You can build
-it like so::
-
-    hpack package.yaml
-    cabal v2-build
-
-You can use ``v2-install`` instead of ``v2-build`` to install the Klister
-executable and associated files to your home directory.
-
-Usage
-=====
-
-Run ``klister repl`` to be dropped into a read-eval-print loop where you can try
-out writing some Klister expressions.
-
-``klister run file.kl`` will evaluate the examples and run the IO
-actions in ``file.kl``.
+Klister [`TyDe 2020`_, `video`_] is a research-stage programming language which combines
```

Comment:
I'd want to call it a "research prototype" here, because it's not really possible to write useful programs in the language outside of the focus (metaprogramming)

Reply:
fixed in 9232b80a847b8c3b51fceeca5bca6c6ab48eeabc